src/mbgl/actor/scheduler.cpp

```diff
@@ -23,6 +24,10 @@ void Scheduler::SetCurrent(Scheduler* scheduler) {
 }
 
 Scheduler* Scheduler::GetCurrent() {
+    if (!localScheduler) {
+        thread_local util::RunLoop runLoop;
+        SetCurrent(&runLoop);
```

Comment:
Does it make sense to do this here or on the caller side? I see RunLoop destructors explicitly setting the current scheduler to nullptr so I'm not sure if it makes sense to have GetCurrent always return a valid instance.

Reply:
I think it makes sense, none of the current call sites assume that `GetCurrent()` ever return `nullptr`.